recipes/mosquitto/build.sh

```diff
@@ -0,0 +1,7 @@
+set -exou
+
+mkdir build
+pushd build
+cmake -D WITH_CJSON=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX ..
```

Comment:
If you cross-compilation to work for `linux_aarch64` you almost certainly need:
```suggestion
cmake $CMAKE_ARGS -DWITH_CJSON=OFF ..
```